src/main/java/network/brightspots/rcv/ContestConfig.java

```diff
@@ -722,30 +721,28 @@ private void validateRules() {
             Logger.log(Level.SEVERE, "batchElimination can't be true in a multi-seat contest!");
           }
         } else { // numberOfWinners == 1
-          if (isMultiSeatSequentialWinnerTakesAllEnabled() ||
-              isMultiSeatBottomsUpUntilNWinnersEnabled() ||
-              isMultiSeatAllowOnlyOneWinnerPerRoundEnabled() ||
-              isMultiSeatAllowMultipleWinnersPerRoundEnabled()) {
+          if (!isSingleWinnerEnabled()) {
             isValid = false;
             Logger.log(
                 Level.SEVERE,
                 "winnerElectionMode can't be \"%s\" in a single-seat contest!",
                 winnerMode.toString()
             );
           }
-
-          if (isHareQuotaEnabled()) {
-            isValid = false;
-            Logger.log(Level.SEVERE, "hareQuota can only be true in a multi-seat contest!");
-          }
         }
       } else { // numberOfWinners == 0
-        if (!isMultiSeatBottomsUpWithThresholdEnabled()
-            || getMultiSeatBottomsUpPercentageThreshold() == null) {
+        if (!isMultiSeatBottomsUpWithThresholdEnabled()) {
           isValid = false;
           Logger.log(Level.SEVERE,
               "If numberOfWinners is zero, winnerElectionMode must be \"%s\" and multiSeatBottomsUpPercentageThreshold must be specified!",
-              winnerMode.toString());
+              WinnerElectionMode.MULTI_SEAT_BOTTOMS_UP_USING_PERCENTAGE_THRESHOLD);
+        } else {
+          if (getMultiSeatBottomsUpPercentageThreshold() == null) {
+            isValid = false;
+            Logger.log(Level.SEVERE,
+                "If winnerElectionMode is \"%s\", multiSeatBottomsUpPercentageThreshold must be specified!",
+                winnerMode.toString());
+          }
         }
       }
     }
```

Comment:
This block here:  if (Utils.isInt(getNumberOfWinnersRaw())) { ... }
Is it doing anything useful?  Should there be an else?

Reply:
If the user provides a non-integer value for this, it will display the message:

> numberOfWinners must be an integer from 0 to 2147483647!

Per a check that's done above. This needs to be here so the remaining validation around that field doesn't occur (and thus throw `NumberFormatException`s) when that's the case.